Remotion/Core/UnitTests/Context/CallContextStorageProviderTest.cs

```diff
@@ -20,6 +20,7 @@
 
 namespace Remotion.UnitTests.Context
 {
+  [Obsolete]
```

Comment:
that test fixture can go bye-bye given the production code is (meant to) throwing exceptions when called.